src/app/components/SEStatusLabel/SEStatusLabel.tsx

```diff
@@ -0,0 +1,96 @@
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+  VoidFunctionComponent,
+} from "react";
+import { ManagedResourceStatus } from "@openapi/generated";
+import { ResourceStatusLabel } from "@app/components/ResourceStatusLabel/ResourceStatusLabel";
+import {
+  ResourceStatus,
+  ResourceStatusDelayed,
+} from "@app/components/ResourceStatusLabel/types";
+
+import { useTranslation } from "react-i18next";
+import { differenceInMinutes } from "date-fns";
+import { usePolling } from "../../../hooks/usePolling/usePolling";
+import { CreationStatusOrder } from "@app/components/SEStatusLabel/SEStatusPopover/types";
+import SEStatusPopover from "@app/components/SEStatusLabel/SEStatusPopover/SEStatusPopover";
+
+export interface SEStatusLabelProps {
+  status: ManagedResourceStatus;
+  resourceType: "bridge" | "processor";
+  requestedAt: Date;
+}
+
+const SEStatusLabel: VoidFunctionComponent<SEStatusLabelProps> = (props) => {
```

Comment:
This is the SE specific status label component. It wraps `ResourceStatusLabel`, mapping SE statuses to the UI ones.

It's also responsible for including the SE specific status popover component when necessary.